### PR TITLE
qa/workunits/objectstore/test_fuse.sh: make test_fuse.sh work with filestore

### PR DESCRIPTION
**Pull request by liewegas**
We skip parts of the test that don't work.

### CODE REVIEW

**Comment by liewegas** [comment]
@tchaikov

**Comment by liewegas** [comment]
(the scrub.py teuthology test doesn't need these parts of the interface.)

File: qa/workunits/objectstore/test_fuse.sh [hunk 1]
```diff
@@ -90,23 +90,29 @@ test ! -s $MNT/meta/all/#-1:7b3f43c4:::osd_superblock:0#/data
...
 # create pg collection
 mkdir --mode 0003 $MNT/0.0_head
-cat $MNT/0.0_head/bitwise_hash_bits
-grep -q 3 $MNT/0.0_head/bitwise_hash_bits
-grep -q 00000000 $MNT/0.0_head/bitwise_hash_start
-grep -q 1fffffff $MNT/0.0_head/bitwise_hash_end
+if [ "$TYPE" = "bluestore" ]; then
+    cat $MNT/0.0_head/bitwise_hash_bits
+    grep -q 3 $MNT/0.0_head/bitwise_hash_bits
+    grep -q 00000000 $MNT/0.0_head/bitwise_hash_start
```
Comment (tchaikov) [inline qa/workunits/objectstore/test_fuse.sh]: "bitwise_hash_start" is always available.

File: qa/workunits/objectstore/test_fuse.sh [hunk 2]
```diff
@@ -90,23 +90,29 @@ test ! -s $MNT/meta/all/#-1:7b3f43c4:::osd_superblock:0#/data
...
+fi
 test -d $MNT/0.0_head/all
 
 mkdir --mode 0003 $MNT/0.1_head
-grep -q 3 $MNT/0.1_head/bitwise_hash_bits
-grep -q 80000000 $MNT/0.1_head/bitwise_hash_start
-grep -q 9fffffff $MNT/0.1_head/bitwise_hash_end
+if [ "$TYPE" = "bluestore" ]; then
+    grep -q 3 $MNT/0.1_head/bitwise_hash_bits
+    grep -q 80000000 $MNT/0.1_head/bitwise_hash_start
```
Comment (tchaikov) [inline qa/workunits/objectstore/test_fuse.sh]: ditto

**Comment by tchaikov** [comment]
other than the nits, lgtm.

**Pull request merged**